admin/src/types.ts

```diff
@@ -42,5 +42,6 @@ export type Session = {
   placesLeft: number;
   dateStart: string | null;
   dateEnd: string | null;
+  canBeDeleted: boolean;
```

Comment:
N'existe pas toujours.
```suggestion
  canBeDeleted?: boolean;
```